prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml

```
@@ -230,6 +230,8 @@ data:
         key: "sys|build"
       - metric_suffix: "config_diskSpace_bytes"
         key: "config|diskSpace"
+      - metric_suffix: "config_hyperthreading_available"
```

```suggestion
      - metric_suffix: "config_hyperthread_available"
```
Would try to stick as close to the original metric as possible